jest.config.js

```diff
@@ -0,0 +1,24 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+
+const { defaults: tsjPreset } = require('ts-jest/presets')
+
+module.exports = {
+  preset: '@shelf/jest-mongodb',
+  transform: tsjPreset.transform,
+  setupFiles: [
+    "<rootDir>/server/test/mongoMemoryServer/databaseSetup.ts"
+  ]
+};
+
+// const tsPreset = require('ts-jest/jest-preset');
```

Comment:
This is needed.... This is another variation of how you could write this file and I will forget where I found it unless I save it here lol....